src/190B20.c

```diff
@@ -1219,9 +1219,43 @@ s32 lookup_defense(DefenseTableEntry* defenseTable, Element elementKey) {
     return normalDefense;
 }
 
-INCLUDE_ASM(s32, "190B20", lookup_status_chance); // exactly (?) the same as lookup_defense
+s32 lookup_status_chance(DictionaryEntry* statusTable, Element statusKey) {
+    DictionaryEntry* row;
+    s32 normalChance = 0;
 
-INCLUDE_ASM(s32, "190B20", lookup_status_duration_mod); // exactly (?) the same as lookup_defense
+    for (row = statusTable; row->key != STATUS_END; row++, statusTable++) {
+        if (row->key == STATUS_DEFAULT) {
+            normalChance = statusTable->value;
+        }
+
+        if (row->key == statusKey) {
+            normalChance = statusTable->value;
+            break;
```

Comment:
Does `return statusTable->value` work here? I think the meaning would be clearer.

Reply:
No that doesn't match